contracts/src/messaging/messenger/CrossChainEnabledObscuro.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity >=0.7.0 <0.9.0;
+
+import "./ICrossChainMessenger.sol";
+
+// TODO: We need to upgrade the open zeppelin version to the 4.x that adds cross chain enabled
+abstract contract CrossChainEnabledObscuro {
```

Comment:
why not "CrossChainEnabled"

The Obscuro bit is confusing me.

Reply:
There is more work needed to link this with OpenZeppelin's CrossChainEnabled. 
If you look at the specializations all of them have the name of the L2 platform following, thus I just stuck with it.
https://docs.openzeppelin.com/contracts/4.x/api/crosschain